curvefs/test/volume/block_device_client_test.cpp

```diff
@@ -325,33 +412,36 @@ TEST_F(BlockDeviceClientTest, TestWriteWithUnAligned) {
         memset(buf, '0', sizeof(buf));
 
         // CASE 1: read failed -> write failed
-        EXPECT_CALL(*fileClient_, Read(10, _, 0, 4096))
+        EXPECT_CALL(*fileClient_, AioRead(10, _, _))
             .WillOnce(Return(-1));
-        EXPECT_CALL(*fileClient_, Write(_, _, _, _))
+        EXPECT_CALL(*fileClient_, AioWrite(_, _, _))
             .Times(0);
         ASSERT_LT(client_->Write(buf, 0, 1), 0);
 
         // CASE 2: read unexpected bytes -> write failed
-        EXPECT_CALL(*fileClient_, Read(10, _, 0, 8192))
-            .WillOnce(Return(8191));
-        EXPECT_CALL(*fileClient_, Write(_, _, _, _))
+        EXPECT_CALL(*fileClient_, AioRead(10, _, _))
+            .WillOnce(Invoke(FakeAioRequest{-1}));
+
+        EXPECT_CALL(*fileClient_, AioWrite(_,  _, _))
             .Times(0);
         ASSERT_LT(client_->Write(buf, 1000, 5000), 0);
 
         // CASE 3: read failed once -> write failed
-        EXPECT_CALL(*fileClient_, Read(10, _, 8192, 4096))
-            .WillOnce(Return(4096));
-        EXPECT_CALL(*fileClient_, Read(10, _, 16384, 4096))
-            .WillOnce(Return(4095));
-        EXPECT_CALL(*fileClient_, Write(_, _, _, _))
+        // EXPECT_CALL(*fileClient_, Read(10, _, 8192, 4096))
```

Comment:
why comment it?

Reply:
removed, we use asynchronous interface now.